LayoutTests/platform/mac/fast/selectors/selection-gap-background-color.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<style>
+::selection {
+    background-color: red;
```

Comment:
Can we add a ref test instead? These png images aren't checked by default whereas ref tests results are compared pixel by pixel by default.

Reply:
I would be happy to, but can you help me understand what a "ref test" is? If you can link to documentation about them or even an example of one then I will try to adapt the current test to that format.

Reply:
See https://docs.webkit.org/Build%20%26%20Debug/Tests.html#reference-tests

Reply:
Thanks! I will look into this and update the PR.

Reply:
I just pushed a new branch with a reftest style test.

Reply:
Hmm, actually there is something wrong with the test I pushed. It's got the wrong background color on the expected.html, but it's still passing. I'll figure it out.

Reply:
All fixed now in the latest push. Fingers crossed!